lib/srv/reexec.go

```diff
@@ -1225,6 +1292,7 @@ func CheckHomeDir(localUser *user.User) (bool, error) {
 		Path: executable,
 		Args: []string{executable, teleport.CheckHomeDirSubCommand},
 		Env:  []string{"HOME=" + localUser.HomeDir},
+		Dir:  string(os.PathSeparator),
```

Comment:
Again, this is misleading and may cause weird problems. Let's just use "/" or a constant.